man/coredns-forward.7

```diff
@@ -56,7 +56,7 @@ forward FROM TO\.\.\. {
     max_fails INTEGER
     tls CERT KEY CA
     tls_servername NAME
-    policy random|round_robin
+    policy random|round_robin|sequential
```

Comment:
this is generated, no need to add this

Reply:
You want me to back it out or are you OK with it as is (since it's just going to get stepped on anyway)?

Reply:
you can leave it